tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1290,10 +1279,12 @@ def check_parameters(self, node):
         3. Missing a return doc in the docstring when a function returns something.
         4. Missing an rtype in the docstring when a function returns something.
         5. Extra params in docstring that aren't function parameters. Change to keywords.
+        6. Misformatted param type in docstring.
 
         :param node: ast.ClassDef or ast.FunctionDef
         :return: None
         """
+        typing_list = ["Optional", "Any", "Union", "List", "Tuple", "Dict", "Iterable", "Iterator"]
```

Comment:
Mapping, MutableMapping, TypedDict, Sequence, Literal, Callable...

Reply:
would be ... mapping, mutuablemapping?, typed dict?, sequence, literal, callable